config/notifiers.go

```diff
@@ -590,11 +589,11 @@ func (c *PushoverConfig) UnmarshalYAML(unmarshal func(interface{}) error) error
 	return nil
 }
 
-// TODO: Move to common?
 
 // SigV4Config is the configuration for signing remote write requests with
 // AWS's SigV4 verification process. Empty values will be retrieved using the
 // AWS default credentials chain.
+// TODO: Move to common.
```

Comment:
Can we make this happen?

Reply:
Once this is in I think we're good to remove that: https://github.com/prometheus/common/pull/309

Reply:
Ok, maybe with this one: https://github.com/prometheus/common/pull/310